common/src/main/java/org/cloudfoundry/autosleep/config/Config.java

```diff
@@ -113,6 +113,12 @@
         String SECRET = "secret";
 
     }
+    
+    interface OrgEnrollmentParameters {
+        enum EnrolledState {
+            backoffice_enrolled, backoffice_opted_out
```

Comment:
Missing the auto_enrolled value documented in https://docs.google.com/document/d/1ZUfs2Okv3Lb4vFpUB0XbRAbKxWhYylNVycKLk2-kB9E/edit#bookmark=id.9a2wsq2xyrtm

``state= auto_enrolled | backoffice_enrolled | backoffice_opted_out``